### PR TITLE
Deal with review comments from #25

### PR DESCRIPTION
**Pull request by timj**
@womullan mainly changes in your section. When I try to cut stuff I'll probably start by moving itemized lists into paragraphs but I'll do that in a separate PR unrelated to #25.

### CODE REVIEW

**Comment by timj** [comment]
I didn't try to define Level 2 WBS since wikipedia article on WBS talks about it and the people that are reading about WBS will know what it means (and I'm tight for space...).

File: agile.tex [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
 
 \subsection{Management}
-The Project Management Guide \cite{DMTN-020} provides  comprehensive details on the mechanisms underpinning LSST Data Management’s approach to project management, the reader is referred to to that document for gory details as required.
-Each institution in the DM team is typically
-responsible for 1 or more Level 2 WBS elements, and each institution has a Technical/Control Account Manager
-(T/CAM) responsible for planning, estimating, monitoring and EVM reporting for that Level 2 WBS element.
-The detailed management plan is in LDM-294\cite{LDM-294}.
+The Project Management Guide \cite{DMTN-020} provides  comprehensive details on the mechanisms underpinning LSST Data Management’s approach to project management, the reader is referred to that document for details as required.
+Each institution in the DM team is typically responsible for one or more Level 2 Work Breakdown Structure (WBS) elements, and each institution has a Technical/Control Account Manager (T/CAM) responsible for planning, estimating, monitoring and Earned Value Management reporting for that Level 2 WBS element.
```
Comment (jdswinbank) [inline agile.tex]: With apologies for the drive by commenting: I think this should be level 3.

To quote DMTN-020:

> The first component is always “1”, referring the LSST Construction Project. “02C” in the second component corresponds to Data Management Construction. Subdivisions thereof are indicated by further digits. Subdivisions at this level correspond to teams within the DM project. [...] These subdivisions are referred to as the third level WBS

Comment (timj) [reply agile.tex]: Thanks. Fixed and I made you a co-author.

File: agile.tex [hunk 2]
```diff
@@ -48,19 +50,18 @@ \subsection{Long Term Planning}
 Planning packages are defined at the fourth level of the Work Breakdown Structure (WBS).
 
 All WBS elements are related to the set of Data Management Products embodied in the System Design \cite{LDM-148}.
-Each product has a product owner to guide the agile development, this is frequently one of the Data Management Scientists.
+Each product has a \emph{product owner} to guide the agile development, this is frequently one of the Data Management Scientists.
 
-During the cycle planning process  effort is drawn from the budget embodied in the planning packages to generate the cycle plan, described in terms of epics in Jira.
-Each epic itself has a particular budget.
+During the planning of each 6 month cycle, effort is drawn from the budget allocated to the planning packages to generate the cycle plan, described in terms of epics in Jira.
+Each epic itself has a particular budget with story points being associated with a specific WBS.
```
Comment (r-owen) [inline agile.tex:56]: This is better but I still find it unclear. I think you are saying two related things:
- Each epic is associated with a WBS element (surely not a WBS?)
- Each epic has a budgeted amount of effort that is drawn from the budget of that WBS element

Consider something like "Each epic is associated with a specific WBS element and its effort (expressed as <i>story points</i>) is drawn from that element's budget".

Refinements:
- If the term "story points" is not used elsewhere you could remove the parenthetical comment.
- "effort" could be expanded, but I find "amount of effort" clumsy; perhaps "budgeted effort" (which adds stutter) or perhaps the parenthetical comment helps?

**Review by r-owen — COMMENTED** [review]
Very nice improvements!

I had just one minor suggested change.

**Pull request merged**